### ISSUE TITLE
No .sketch file

### ISSUE THREAD

**Issue by Scrivs** [mheesakkers/sketch-plugin-metaball#14]
I don't see the metaball.sketchplugin file when opening the ZIP.

**Comment by mheesakkers** [comment]
It seems to work on my end. You're using this URL? :
https://github.com/mheesakkers/sketch-plugin-metaball/releases/latest/download/metaball.sketchplugin.zip

**Comment by Scrivs** [comment]
Yep. Not sure if my computer (Mac) is hiding the file, but it's not showing up when I unzip.

**Comment by mheesakkers** [comment]
And you also have Sketch installed?
